extensions/ql-vscode/src/remote-queries/view/RemoteQueries.tsx

```diff
@@ -369,7 +370,7 @@ export function RemoteQueries(): JSX.Element {
     return <div>Waiting for results to load.</div>;
   }
 
-  const showAnalysesResults = false;
+  const showAnalysesResults = true;
```

Comment:
Optional: Change this back to false so I don't get merge conflicts 😂

Reply:
very sensible 😁